### DISCUSSION TITLE
Expose Node HW Configuration

### DISCUSSION THREAD

**Discussion by papperone** [dougiteixeira/proxmoxve#184] [Feature request] [answered]
I would like to have for each of my node this info available on a text sensor; this is already retrieved somehow by the integration as it's shown on the page (see picture below) but I cannot find it in the many entity exposed; will it be possible to have this cahnge implemented? or it's too complex...
![368065002_1107939543919891_4225440357865322140_n](https://github.com/dougiteixeira/proxmoxve/assets/28104633/b0aa188e-4569-4f8e-9356-ea3726fa1ae1)

**Comment by dougiteixeira** [comment, marked as answer]
This is a fixed property of the device, I don't see much use in having this as a sensor.

In specific cases, you can use template to obtain this value, for example:

```
{{ device_attr('sensor.node_pve_status', 'model') }}
```

I believe the biggest use case is in Dashboards, often you can do this without creating a sensor, but if you want you can create a sensor template with the code above.

You can see more use cases for device templates in the [Home Assistant documentation](https://www.home-assistant.io/docs/configuration/templating/#devices).

**Reply by papperone** [reply]
thanks, that's more than ok for my need (indeed populate a dashboard with all my nodes details)...
where can I find the list of all attributes available (next one is ProxMox version and would be great to check what other is avail)?

Regards

**Reply by dougiteixeira** [reply]
The list of device attributes can be consulted in the [developer documentation (link)](https://developers.home-assistant.io/docs/device_registry_index#device-properties).


However, not all of them apply to devices with this integration.
